.github/workflows/nightly.yml

```diff
@@ -36,6 +36,29 @@ jobs:
           mvn archetype:generate -DinteractiveMode=false -DaskForDefaultPropertyValues=false -DarchetypeGroupId=org.eclipse.starter -DarchetypeArtifactId=jakarta-starter -DarchetypeVersion=2.3.0-SNAPSHOT -DjakartaVersion=8 -Dprofile=web -DjavaVersion=8 -DoutputDirectory=app -Dgoals="clean package"
           rm -rf app
 
+
+      - name: Run Archetype for EE 8 Core Profile, SE 8
+        id: ee_8_core_se_8_none
+        continue-on-error: true
+        run: |
+          mvn archetype:generate -DinteractiveMode=false -DaskForDefaultPropertyValues=false -DarchetypeGroupId=org.eclipse.starter -DarchetypeArtifactId=jakarta-starter -DarchetypeVersion=2.3.0-SNAPSHOT -DjakartaVersion=8 -Dprofile=core -DjavaVersion=8 -DoutputDirectory=app -Dgoals="clean package" > mvn_output.txt 2>&1
+
+      - name: Check Maven Build Status
```

Comment:
Please see if both of the following checks can be combined. From a business logic and readability perspective, they are inseparable. We are checking that the build fails for a specific reason.